### DISCUSSION TITLE
How to implement lexical floating tool bar

### DISCUSSION THREAD

**Discussion by danishsjjd** [mdx-editor/editor#399] [Q&A]
Hey,

I was trying to create a floating input like the one we have in [lexical playground](https://playground.lexical.dev/)

![image](https://github.com/mdx-editor/editor/assets/92915951/e3619b23-b70d-49ab-92fa-4c46c61cd7c2)

I'm having hard time implementing this, so a question arises in my mind: why not just ask the maintainer? Could you please point out to me whether it's something I can integrate easily or if I need to dig more into the code, as I haven't played with it much?

Thanks.

**Comment by arshad-yaseen** [comment]
+1 @petyosi
